kubelet/datadog_checks/kubelet/common.py

```diff
@@ -88,19 +105,34 @@ def __init__(self, podlist):
                     short_cid = cid.split("://", 1)[-1]
                     self.containers[short_cid] = ctr
 
-    def is_excluded(self, cid):
+    def is_excluded(self, cid, pod_uid=None):
         """
         Queries the agent6 container filter interface. It retrieves container
         name + image from the podlist, so static pod filtering is not supported.
+
+        Result is cached between calls to avoid the python-go switching cost for
+        prometheus metrics (will be called once per metric)
```

Comment:
curiosity: what's the cost of that python-go call ?

Reply:
I have no numbers yet (we should benchmark that if you wanna give it a go!), but we need to copy the two strings twice + go through the golang scheduler